src/cartservice/cartservice.csproj

```diff
@@ -9,9 +9,13 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Grpc.AspNetCore" Version="2.37.0" />
+    <PackageReference Include="Grpc.AspNetCore" Version="2.34.0" />
+    <PackageReference Include="OpenTelemetry" Version="1.1.0-beta1" />
+    <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.1.0-beta1" />
+    <PackageReference Include="OpenTelemetry.Extensions.Hosting" Version="1.0.0-rc3" />
+    <PackageReference Include="OpenTelemetry.Instrumentation.AspNetCore" Version="1.0.0-rc3" />
     <PackageReference Include="StackExchange.Redis" Version="2.2.4" />
-    <PackageReference Include="Grpc.HealthCheck" Version="2.37.1" />
+    <PackageReference Include="Grpc.HealthCheck" Version="2.36.1" />
```

Comment:
Is there a reason this is going backwards?

Reply:
There was a bug in a release of OpenTelemetry's C# sdk that required rolling back some dependencies.  It has since been resolved.  I advanced this (and a couple others) to their latest minor versions.